CHANGELOG.md

```diff
@@ -1,3 +1,41 @@
+# 92.2.0
+
+This minor release adds new features to MDCAlertController and MDCTabBarView.
+
+## New features
+
+MDCAlertController has a new accessoryView property. This property can be used to add custom views
```

Comment:
Nit: Please put `accessoryView` in code markers (``) to identify it as an API.

Reply:
Done.